build.gradle

```diff
@@ -26,17 +25,19 @@ versionFile.mkdirs()
 versionFile.delete()
 versionFile << version
 
+final def internalRepoHost = System.getenv('SNPS_INTERNAL_ARTIFACTORY')
+
 repositories {
+    maven { url "${internalRepoHost}/artifactory/jcenter" }
```

Comment:
These lines are related to our short-term fix that allows the magpie library to be fetched from internal artifactory.